ok-marketplace-be/ok-marketplace-repo-cassandra/src/test/resources/docker-compose-cs.yml

```
@@ -0,0 +1,36 @@
+version: '3.9'
+services:
+  cassandra:
+    image: cassandra:5.0.3
+    ports:
+      - "9042:9042"
```

Из-за этого может быть ошибка. Порт может быть уже занят другим экземпляром.
Нужно делать здесь случайный порт
```suggestion
      - "9042"
```

---

не помогло